### DISCUSSION TITLE
plot-correlator not supporting csv files from nsight profiler

### DISCUSSION THREAD

**Discussion by QuangAnh-Hoang** [accel-sim/accel-sim-framework#243] [Q&A]
Hi,
I've been trying to replicate the correlation results for RTX 3070, which requires nsight CLI profiler to be used. However, as I tried to plot the correlation between simulated data vs. hardware data, it seems like plot-correlation.py does not support the output format of nsight, especially with [plot-correlation.py:508](https://github.com/accel-sim/accel-sim-framework/blob/2260456ea5e6a1420f5734f145a4b7d8ab1d4737/util/plotting/plot-correlation.py#L508). Since I'm not 100% sure if this problem has been handled somewhere else or not, I would like to ask for clarification and probably what options I may have to replicate the experiment. 
Thanks

**Comment by JRPan** [comment]
What Nsight Compute version are you using? Maybe the formatting changed in the latest profiler.

**Reply by QuangAnh-Hoang** [reply]
I'm running CUDA 11.4 and it comes with nsight compute 2021.2.0

**Reply by JRPan** [reply]
That should work with the script. Can you check if the hardware data are collected correctly? There should be several CSV files under hw_run folder.
